### PR TITLE
Added new connector Asite Hong Kong and Asite Canada

### PR DESCRIPTION
**Pull request by asitedeveloper**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/Asite (Canada)/Readme.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# About
+
+Asite is an open construction platform that enables organizations with comprehensive range of solutions connect dispersed teams across the lifecycle of capital assets by collaborate, plan, design, and build with seamless information sharing across the entire supply chain which helps capital project owners stay at the forefront of innovation, maintaining a golden thread of information.
```
Comment (vmanoharas) [inline certified-connectors/Asite (Canada)/Readme.md:2]: Please update readme file as per the template in this link. 
https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

File: certified-connectors/Asite (Canada)/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "{{ client id }}",
+          "scopes": [
+            "DOCUMENT_READ DOCUMENT_WRITE"
+          ],
+          "redirectMode": "Global",
```
Comment (vmanoharas) [inline certified-connectors/Asite (Canada)/apiProperties.json]: Please modify redirctMode as "GlobalPerConnector" instead "Global"

File: certified-connectors/Asite (Hong Kong)/Readme.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# About
+
+Asite is an open construction platform that enables organizations with comprehensive range of solutions connect dispersed teams across the lifecycle of capital assets by collaborate, plan, design, and build with seamless information sharing across the entire supply chain which helps capital project owners stay at the forefront of innovation, maintaining a golden thread of information.
```
Comment (vmanoharas) [inline certified-connectors/Asite (Hong Kong)/Readme.md:2]: Please modify readme.md file as per the template in the below url. 
https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

File: certified-connectors/Asite (Hong Kong)/apiProperties.json [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "{{ client id }}",
+          "scopes": [
+            "DOCUMENT_READ DOCUMENT_WRITE"
+          ],
+          "redirectMode": "Global",
```
Comment (vmanoharas) [inline certified-connectors/Asite (Hong Kong)/apiProperties.json]: Please modify redirectMode as "GlobalPerConnector" instead Global.

**Review by vmanoharas — COMMENTED** [review]
Please resolve review comments

File: certified-connectors/Asite (Canada)/Readme.md (hunk 1 above)
Comment (asitedeveloper) [reply certified-connectors/Asite (Canada)/Readme.md:2]: @vmanoharas I have changed as per this template link.

File: certified-connectors/Asite (Hong Kong)/Readme.md (hunk 3 above)
Comment (asitedeveloper) [reply certified-connectors/Asite (Hong Kong)/Readme.md:2]: @vmanoharas I have changed as per this template link.

File: certified-connectors/Asite (Canada)/apiProperties.json (hunk 2 above)
Comment (asitedeveloper) [reply certified-connectors/Asite (Canada)/apiProperties.json]: @vmanoharas I have changed as per your suggestion.

File: certified-connectors/Asite (Hong Kong)/apiProperties.json (hunk 4 above)
Comment (asitedeveloper) [reply certified-connectors/Asite (Hong Kong)/apiProperties.json]: @vmanoharas I have changed as per your suggestion.

File: certified-connectors/Asite (Canada)/apiProperties.json (hunk 2 above)
Comment (vmanoharas) [reply certified-connectors/Asite (Canada)/apiProperties.json]: @asitedeveloper, still the redirectMode is "Global" please change as "redirectMode": "GlobalPerConnector"

File: certified-connectors/Asite (Hong Kong)/apiProperties.json (hunk 4 above)
Comment (vmanoharas) [reply certified-connectors/Asite (Hong Kong)/apiProperties.json]: @asitedeveloper, still the redirectMode is "Global" please change as "redirectMode": "GlobalPerConnector"

**Review by vmanoharas — COMMENTED** [review]
Please resolve comments.

File: certified-connectors/Asite (Canada)/apiProperties.json (hunk 2 above)
Comment (asitedeveloper) [reply certified-connectors/Asite (Canada)/apiProperties.json]: image
I have already changed "GlobalPerConnector" in this file but I don't how why it is not reflecting.
Please refer attached screenshot.
commit Id : 43b4415a16384e53b90c20df6b5f57500a73151d

File: certified-connectors/Asite (Hong Kong)/apiProperties.json (hunk 4 above)
Comment (asitedeveloper) [reply certified-connectors/Asite (Hong Kong)/apiProperties.json]: image

I have already changed "GlobalPerConnector" in this file but I don't how why it is not reflecting.
Please refer attached screenshot.
commit Id : 43b4415a16384e53b90c20df6b5f57500a73151d

File: certified-connectors/Asite (Canada)/apiProperties.json (hunk 2 above)
Comment (vmanoharas) [reply certified-connectors/Asite (Canada)/apiProperties.json]: Thank you @asitedeveloper

**Comment by vmanoharas** [comment]
Hi @asitedeveloper, 
there is an issue in our validation checker, we are actively working on to resolve the same. We will approve your PR once it is resolved. Apologies and thanks for your patience and understanding.

**Comment by asitedeveloper** [comment]
@vmanoharas Can you please give me a tentative time for approve this PR?

**Comment by vmanoharas** [comment]
Hello @asitedeveloper, 

Please accept my apology, I will approve this PR now.  Thanks for your patience and understanding.

**Review by vmanoharas — APPROVED** [review]
Hello @asitedeveloper 
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**